atomic_defi_design/Dex/Settings/RecoverSeedModal.qml

```diff
@@ -16,34 +16,37 @@ MultipageModal
     property var portfolio_model: API.app.portfolio_pg.portfolio_mdl
     property var settings_page: API.app.settings_pg
 
-    property bool wrongPassword: false
+    property bool _isPasswordWrong: false
 
     function tryViewKeysAndSeed()
     {
         if (!submitButton.enabled) return
 
-        const result = API.app.settings_pg.retrieve_seed(API.app.wallet_mgr.wallet_default_name, inputPassword.field.text)
+        const result = API.app.settings_pg.retrieve_seed(API.app.wallet_mgr.wallet_default_name, _inputPassword.field.text)
 
         if (result.length === 2)
         {
             seedLabel.text = result[0]
             rpcPwLabel.text = result[1]
-            wrongPassword = false
+            _isPasswordWrong = false
             root.nextPage()
             loading.running = true
         }
         else
         {
-            wrongPassword = true
+            console.log("Incorrect password!")
```

Comment:
No need to log here: interaction already diplsayed on the UX

Reply:
removed